src/runtime/RegExpObject.cpp

```diff
@@ -497,4 +498,14 @@ void RegExpObject::pushBackToRegExpMatchedArray(ExecutionState& state, ArrayObje
         }
     }
 }
+
+bool RegExpObject::checkMatchSymbol(ExecutionState& state)
+{
+    ObjectGetResult symbol = get(state, ObjectPropertyName(state, state.context()->vmInstance()->globalSymbols().match));
+    Value val = symbol.value(state, Value(this));
+    if (!val.isUndefined()) {
+        return val.toBoolean(state);
+    }
+    return true;
```

Comment:
At this point, I think that `false` should be returned according to the stardard as follow.
>  If Type(argument) is not Object, return false.
    Let isRegExp be Get(argument, @@match).
    ReturnIfAbrupt(isRegExp).
    If isRegExp is not undefined, return ToBoolean(isRegExp).
    If argument has a [[RegExpMatcher]] internal slot, return true.
    Return false.